go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/devfile/api/v2
 
-go 1.21
+go 1.22.0
```

Comment:
@rohanKanojia Given we have the intention to update to Go 1.24 #1701 is it possible to do that here? Or do you need a Go 1.22 reference?

Reply:
In [DevWorkspaceOperator](https://github.com/devfile/devworkspace-operator/blob/main/go.mod#L3) we use go 1.23.0 . Is it okay if I bump it to go 1.23 instead?

Reply:
While trying to bump the version, I noticed that even latest version of universal developer image [quay.io/devfile/universal-developer-image:ubi9-latest](https://quay.io/repository/devfile/universal-developer-image) uses go 1.22

https://github.com/devfile/api/blob/30244664344292f45887fe00f6dc31a1e74132c3/.devfile.yaml#L24

Reply:
I've updated the Go version to 1.23 and the UDI version to the latest available version (which, however, doesn't support Go 1.23 yet). Let me know if it's okay.

Reply:
@rohanKanojia Seems like there is some work in getting the Go version updated, lets revert back to 1.22 to meet the requirement of these changes and we'll deal with the bump to 1.24 as part of #1701 as a separate effort.